design/sketch.webidl

```diff
@@ -212,17 +212,16 @@ interface WebGPUPipelineLayout {
 };
 
 // BindGroup
-dictionary WebGPUBufferBinding {
-    WebGPUBuffer buffer;
-    u32 offset;
-    u32 size;
-};
-
-typedef (WebGPUSampler or WebGPUTextureView or WebGPUBufferBinding) WebGPUBindingResource;
-
 dictionary WebGPUBinding {
     uint32_t binding;
```

Comment:
I don't think we have `uint32_t` in IDL, do we?
Let's use `u32` consistently as in other places.

Reply:
Done